src/manifest/adaptation.ts

```diff
@@ -40,14 +40,17 @@ export interface IRepresentationInfos { bufferType: IAdaptationType;
                                         isClosedCaption? : boolean;
                                         normalizedLanguage? : string; }
 
-export type IRepresentationFilter = (representation: Representation,
-                                     adaptationInfos: IRepresentationInfos) => boolean;
+export type IRepresentationFilter = (
+  representation: Representation,
+  adaptationInfos: IRepresentationInfos
+) => boolean;
 
 export interface IAdaptationArguments { id : string;
                                         representations : IRepresentationArguments[];
                                         type : IAdaptationType;
 
                                         // -- optional
+                                        isTrickModeFor? : string;
```

Comment:
As said earlier, might be more logical to already have the trickModeTrack property here